lib/index.js

```diff
@@ -198,6 +207,7 @@ EslintValidationFilter.prototype.postProcess = function postProcess(results /* ,
  */
 EslintValidationFilter.create = function(inputNode, options) {
   options = options || {};
+  options[FACTORY_METHOD_USED] = true;
```

Comment:
This leaks the symbol (since it is added to the object passed in). An alternate that doesn't leak the symbol would be:

```js
.create = function(inputNode, _options) {
  let options = Object.assign({}, _options, {
    [FACTORY_METHOD_USED]: true
  });
}
```

Reply:
good point, though I doubt anyone would choose this workaround if the proper solution is so much easier 😉

Reply:
fixed